### DISCUSSION TITLE
Load two fonts simultaneously

### DISCUSSION THREAD

**Discussion by davorvr** [Bodmer/TFT_eSPI#1277] [Q&A - Fonts] [answered]
Hello, I'd like to use multiple font sizes with smooth fonts. Is there a way to simultaneously have two fonts loaded instead of loading-unloading whenever I need to switch? Thanks for the awesome library!

**Comment by en-ot** [comment, marked as answer]
Hi!
You can try my version of this lib (https://github.com/en-ot/TFT_eSPI.git)
It can not load two fonts simultaneously, but switching between fonts can be faster than with original lib, because no font scanning is performed on loadFont, Also no memory allocation-deallocation is used to load smooth font. But You have to convert your fonts to use it.
